tracer/test/test-applications/integrations/Samples.HttpMessageHandler/Program.cs

```diff
@@ -38,15 +40,15 @@ public static async Task Main(string[] args)
                 Console.WriteLine("Sending async request with default HttpClient.");
                 using (var client = new HttpClient())
                 {
-                    await RequestHelpers.SendHttpClientRequestsAsync(client, tracingDisabled, Url, RequestContent);
+                    RequestHelpers.SendAsyncHttpClientRequests(client, tracingDisabled, Url, RequestContent);
```

Comment:
Aren't we losing coverage on correctly awaiting these?
Would it make sense to add a single call for sync over async instead of converting all?

Reply:
As long as we're not using custom awaiters in our instrumentation code, we're not losing any coverage